specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -2132,6 +2132,16 @@
         }
       }
     },
+    "EncryptionIdentity":{
```

Comment:
Please add a example.
The example should be get from server, to be aligned with server behavior.

Reply:
Sorry for the delays on this. I had to attend to other higher priority tasks. I will update with examples soon.